src/index.html

```diff
@@ -12,11 +12,14 @@
 
 <body>
     <h1>a-MAZE-ing</h1>
-    <button class="moveForward">⬇️</button>
-    <button class="rotate">↩️</button>
-    <button class="execute">✅</button>
+    <h3 class="missionControl-title">Mission Control</h3>
+    <div class="missionControl">
+        <button class="moveForward">⬇️</button>
```

Comment:
Can this use something like font awesome to make the icons a little bigger? Its not required tho

Reply:
🤯 I didn't know about awesome. Let's add them! I'll share the new icons with you first, then I'll implement them. Thanks!

Reply:
Awesome, not required for this PR, don't wanna hold you back